### DISCUSSION TITLE
Using JOOHOI DNS Credentials

### DISCUSSION THREAD

**Discussion by bagelwoof** [0xJacky/nginx-ui#409] [Q&A]
I'm a little lost trying to set up credentials for JOOHOI's ACME-DNS. 

I used docker compose to install the latest release, and setting up these credentials is one of the first things I'm doing.

While the API docs provide terse descriptions of ACME_DNS_API_BASE and ACME_DNS_STORAGE_PATH, there are no examples of what those inputs should look like. 

Has anyone set up JOOHOI's ACME-DNS in this app?
Can you provide an example of the credentials?
Does the container use LEGO?

**Comment by One-Regular-Guy** [comment]
Did you find out how to do this?
